File: HELLO_PYTHON/day06/mysql_insert.py
import pymysql

conn = pymysql.connect(host='localhost', user='root',port=3305, password='python', db='python', charset='utf8')
 
curs = conn.cursor(pymysql.cursors.DictCursor)

sql = 'select * from emp'
# sql1 = 'insert into emp(e_id, e_name, gen, addr) values(%s,%s,%s,%s)'
sql2 = "insert into emp(e_id, e_name, gen, addr) values(5,'5','5','5')"
# val = (4,"4","4","4")

curs.execute(sql)
# curs.execute(sql,(4,4,4,4))

table = curs.fetchall()

# print(table)

conn.commit()

for a in table:
    print(a)

curs.close()
conn.close()

File: HELLO_NODE2/mysql_insert.js
const mysql  = require('mysql');
const connection = mysql.createConnection({
  host		: "localhost", // 데이터베이스 주소
  port		: "3305", // 데이터베이스 포트
  user		: "root", // 로그인 계정
  password	: "python", // 비밀번호
  database	: "python" // 엑세스할 데이터베이스
});

connection.connect();
var sql='INSERT INTO emp (e_id,e_name,gen,addr) VALUES(?,?,?,?)';
var param=[1,'1','1','1'];

connection.query(sql,param, (error, result) => {
	 console.log(result.affectedRows)
});

connection.end();